frontend/src/dashboard/JobList.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+    <div v-if="isAuthentificated">
+        <h3 class="is-size-5">Your active jobs</h3>
+        <div v-for="job in jobList" :key="job['job_id']">
+            <router-link class="message" :to="{ name: 'job', query: { id: job['job_id'] }}">
+                <div class="message-header">
+                    <b-icon v-bind="iconAttributesForState(job['state'])"></b-icon>
+                    {{job['job_id']}}
+                </div>
+                <div class="message-body has-text-left">
+                    <div>
+                        <span class="has-text-weight-semibold">Name</span>
+                        {{job['name']}}
+                    </div>
+                    <div>
+                        <span class="has-text-weight-semibold">Created</span>
+                        {{job['created']}}
+                    </div>
+                </div>
+            </router-link>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+import axios from "axios";
+
+@Component
+export default class JobList extends Vue {
+    jobList: object[] = [];
```

Comment:
Sollen wir an der Stelle eine Domain-Klasse `Job` einführen? Das generische `object` ist irgendwie nicht im Sinne der Typsicherheit. Andererseits müsste man dann irgendwie beim parsen der response darauf mappen ... was meint ihr?

Reply:
Die Diskussion hatten Richard und ich auch in etwa beim Mittagessen: Wie unterschiedlich sind die Jobs am Ende? Ich denke es macht aber auf jeden Fall Sinn, da eine Domain-Klasse einzuführen bzw. für die verschiedenen Jobs Domain Klassen.